FightPandemics/Features/TabBar/RootTabBarController.swift

```diff
@@ -59,6 +59,7 @@ final class RootTabBarController: UITabBarController {
         super.viewDidLoad()
 
         customizeTabBar()
+        // self = <FightPandemics.RootTabBarController: 0x7f86d782ac00>
```

Comment:
If I were submitting this for production, I'd remove the comment. I'm leaving it here so that it's clear which instance was crashing and which instance was wrong.